tests/reg_tests/test_actuator.py

```diff
@@ -252,9 +250,7 @@ def test_actuator_thrust_and_heat(self):
         az_heat = 1e5
         self.ap.setDesignVars({"thrust": az_force, "heat": az_heat})
 
-        # We dont need to rerun, restart file has the correct state. just run a residual
-        # self.CFDSolver(self.ap)
-        self.CFDSolver.getResidual(self.ap)
+        self.CFDSolver(self.ap)
```

Comment:
Now that you switched to `__call__()` we can use `assert_solution_failure()` here and in the following sections as well!